deploy/firewall-controller.yaml

```diff
@@ -103,4 +233,948 @@ roleRef:
   name: firewall-controller
 subjects:
 - kind: User
-  name: system:serviceaccount:firewall-controller:default
+  name: system:serviceaccount:firewall:default
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: evebox-agent-config
+  namespace: firewall
+data:
+  agent.yaml: |
+    server:
+      url: http://evebox-server-service:80
+    bookmark-directory: "/var/lib/evebox"
+    input:
+      filename: "/var/log/suricata/eve.json"
+      custom-fields:
+        host: "evebox-agent"
+        clusterid: "firewall-controller-test"
+        projectid: "development"
+    rules:
+      - /etc/suricata/rules/*.rules
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: evebox-server-config
+  namespace: firewall
+data:
+  evebox.yaml: |
+    data-directory: /var/lib/evebox
+    http:
+      tls:
+        enabled: false
+    # Database configuration.
+    database:
+      # Database type: elasticsearch, sqlite.
+      type: sqlite
+    authentication:
+      required: no
+      # Type of login required:
+      # - username         -- just a username...
+      # - usernamepassword -- username and password
+      # env: EVEBOX_AUTHENTICATION_TYPE
+      type: usernamepassword
+
+      # A little message that is displayed in the login dialog.
+      #login-message: Some message here...
+    input:
+      enabled: false
+      rules:
+        - /etc/suricata/rules/*.rules
+    geoip:
+      disabled: true
+    # Event services: links that will be provided on events to link to additonal
+    # services.
+    event-services:
+
+      # Custom service to link the rule in Scirius.
+      - type: custom
+        enabled: false
+        name: Scirius
+
+        # Only make available for alert types.
+        event-types:
+          - alert
+
+        # URL template. All eve values can be used.
+        url: https://10.16.1.179/rules/rule/{{alert.signature_id}}
+
+      # Custom service to link to Dumpy for full packet capture.
+      #
+      # This one has no event-types meaning its available for all event types.
+      - type: custom
+        enabled: false
+        name: Dumpy
+
+        # The URL template, {{raw}} expands to the raw eve event as a JSON
+        # string which is then url encoded.
+        url: http://10.16.1.1:7000/?event={{raw}}
+
+        # Open in new window. The default is the same window.
+        target: new
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: suricata-config
+  namespace: firewall
+data:
+  suricata.yaml: |
+    %YAML 1.1
+    ---
+
+    vars:
+      # more specific is better for alert accuracy and performance
+      address-groups:
+        HOME_NET: "[192.168.0.0/16,10.0.0.0/8,172.16.0.0/12]"
```

Comment:
could this be filled with internal prefixes + trusted networks?

Reply:
This another example of how we spread configurations between metal-networker and firewall-controller. This file is written by metal-networker, so actually no possibility to access these variables.
This file is only a example deployment for local development in kind. We need to think about a easier solution here.